support_bundle/get_support_bundle.sh

```diff
@@ -161,7 +161,11 @@ main() {
         curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/v2/users/light" >> ${LOG_DIR}/users.json
         curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/v2/teams/light" >> ${LOG_DIR}/teams.json
 
-        TO_EPOCH_TIME=$(gnudate -d "$(gnudate +%H):00:00" +%s)
+        if [[ $OSTYPE == 'darwin'* ]]; then
+            TO_EPOCH_TIME=$(date -j -f "%a %b %d %T %Z %Y" "`date`" "+%s")
+        else
+            TO_EPOCH_TIME=$(gnudate -d "$(gnudate +%H):00:00" +%s)
```

Comment:
I think at this point, we can probably get rid of the `gnudate` function as a whole and just leverage `date`.

cc: @patrickhargett

Reply:
date -d option is not available in mac, 
I think we will still need the OS check

Reply:
got it, removed gnudate and used date instead. confirmed I receive the same result for linux and macos.